configuration/security.md

```diff
@@ -218,20 +296,33 @@ Limit login attempts on ssh port using tcp: this denies connection if an IP addr
 sudo ufw limit ssh/tcp
 ```
 
-Deny access to port 30 from IP address 192.168.2.1
+Deny incoming requests from IP address 192.168.2.1 to port 22 only
```

Comment:
I've never set up a firewall myself (so apologies of this is a naive question), but if you're blocking a specific IP address, isn't it more likely that you'd block access to all ports, rather than just a specific port? :man_shrugging:

Reply:
Yes, it's actually redundant if you're blocking all incoming requests by default already but this is just an example of what's possible. This is why the other rules for allowing and limiting are included as well.